zerver/webhooks/buildbot/doc.md

```diff
@@ -0,0 +1,11 @@
+Zulip supports integration with CircleCI and can notify you of
+your build statuses.
+
+1. {!create-stream.md!}
+
+2. {!create-bot-construct-url-indented.md!}
+
+3. Add a new Zulip reporter in your configuration file with the endpoint set to your
+ zulip server and token set to the bot's api key
```

Comment:
Could you please add a bit more details to this section? Perhaps a link to docs that explain how to add a new Zulip reporter would be helpful as well. Also:
* `zulip` should be capitalized.
* `api key` should be `API key`.